apps/system/locales/system.en-US.properties

```diff
@@ -477,7 +477,10 @@ payment-flow=Purchase
 
 # Remote Debugger Connection Dialog
 remoteDebuggerPromptUSB=Allow USB debugging connection?
-remoteDebuggerPromptTCP=Allow remote debugging connection from {{host}}:{{port}}?
+remoteDebuggerPromptTCP.innerHTML=Allow remote debugging connection from {{host}}:{{port}}?<br/><br/>Client ID: {{cert}}
```

Comment:
I'll probably change `Client ID:` to `Their Cert:`. On @past's advice from Gecko changes, we've avoided saying the words "client" or "server" to the user so far.